fs/config.go

```diff
@@ -87,6 +87,7 @@ type ConfigInfo struct {
 	StreamingUploadCutoff  SizeSuffix
 	StatsFileNameLength    int
 	AskPassword            bool
+	PasswordCommand        string
```

Comment:
I think it would be best to use `SpaceSepList` here instead of string.

The parser will then break it up into space separated parts, but also allow you to use " etc so you can have parameters with spaces in.  It uses the escaping rules from [encoding/csv](https://golang.org/pkg/encoding/csv/)